src/main/resources/application.properties

```diff
@@ -1,5 +1 @@
-spring.datasource.driver-class=com.mysql.jdbc.Driver;
-spring.datasource.url=jdbc:mysql://localhost:3306/jora
-spring.datasource.username=root
-spring.datasource.password=password
 server.port=8081
```

Comment:
just confirming, the port doesn't matter right? it can just be whatever we want?

Reply:
I don't think it does, but 8081 will probably be better in the long run